common/test/acceptance/tests/lms/test_lms_progress.py

```diff
@@ -0,0 +1,178 @@
+
+    def test_passing_info_table_is_visible(self):
+        """
+        Ensures that the passing information table is visible when at least one passing grade
+        is greater than 0 and it does not dispaly Assignments with 0 passing grade.
+        """
+        self.set_assignments([
+            {
+                'type': 'Homework', 'passing_grade': 70, 'weight': 50,
+                'min_count': 2, 'drop_count': 0, 'short_label': 'HW',
+                'passing_grade_enabled': True
+            },
+            {
+                'type': 'Exam', 'passing_grade': 0, 'weight': 50,
+                'min_count': 1, 'drop_count': 0, 'short_label': 'EX',
+                'passing_grade_enabled': True
+            },
+        ])
+
+        self.progress_page.visit()
+        self.assertTrue(self.progress_page.has_passing_information_table)
+        self.assertEqual(
+            self.progress_page.passing_information_table.status,
+            [('Homework', '70', '0', 'Not pass')]
+        )
+
+    def test_passing_info_table_has_correct_values(self):
+        """
+        Scenario: Ensures that the passing information table has correct values
+        Given I have a course with 2 categories ("Homework", "Exam")
+        And "Homework" has passing grade "70" and contains 2 problems
+        And "Exam" has passing grade "0" and contains 1 problem
+        When I pass the 1st Homework's problem (current grade for the category is 50)
+        And I go to the Progress page
+        Then I see that "Homework" category is not passed in the passing information table
+        When I pass the 2nd Homework's problem (current grade for the category is 100)
+        And I go to the Progress page
+        Then I see that "Homework" category is passed in the passing information table
+        """
+        self.set_assignments([
+            {
+                'type': 'Homework', 'passing_grade': 70, 'weight': 50,
+                'min_count': 2, 'drop_count': 0, 'short_label': 'HW',
+                'passing_grade_enabled': True
+            },
+            {
+                'type': 'Exam', 'passing_grade': 0, 'weight': 50,
+                'min_count': 1, 'drop_count': 0, 'short_label': 'EX',
+                'passing_grade_enabled': True
+            },
+        ])
+
+        self.check_problem('Test Section 1', 'Test Subsection 1', answer='6.5')
+        self.progress_page.visit()
+        self.assertTrue(self.progress_page.has_passing_information_table)
+        self.assertEqual(
+            self.progress_page.passing_information_table.status,
+            [('Homework', '70', '50', 'Not pass')]
+        )
+
+        self.check_problem('Test Section 1', 'Test Subsection 2', answer='6.5')
+        self.progress_page.visit()
+        self.assertTrue(self.progress_page.has_passing_information_table)
+        self.assertEqual(
+            self.progress_page.passing_information_table.status,
+            [('Homework', '70', '100', 'Pass')]
+        )
```

Comment:
> Does it make sense to add one more test that:
> 1. enable passing grade
> 2. check that the section is not passed with passing grade [('Homework', '70', '50', 'Not pass')] .
> 3. disable passing grade for the section
> 4. check that the section IS PASSED.
> 
> What do you think?

@polesye 
UPDATE: this test is not valid in our current implementation - when `passing_grade_enabled` is not checked then row for such Section is not presented in the passing grade table.
<img width="562" alt="screen shot 2016-03-18 at 11 50 17 pm" src="https://cloud.githubusercontent.com/assets/877401/13893082/54f9fe6a-ed64-11e5-8014-2fb0e5660dad.png">
<img width="311" alt="screen shot 2016-03-18 at 11 50 07 pm" src="https://cloud.githubusercontent.com/assets/877401/13893081/54f7e3c8-ed64-11e5-81f8-8b7a5647584a.png">